app/src/main/java/org/oppia/android/app/devoptions/vieweventlogs/EventLogItemViewModel.kt

```diff
@@ -0,0 +1,34 @@
+package org.oppia.android.app.devoptions.vieweventlogs
+
+import org.oppia.android.app.model.EventLog
+import org.oppia.android.app.viewmodel.ObservableViewModel
+import org.oppia.android.util.system.OppiaDateTimeFormatter
+import javax.inject.Inject
+
+/** [ViewModel] for displaying a event log. */
+class EventLogItemViewModel @Inject constructor(
+  val eventLog: EventLog,
+  private val oppiaDateTimeFormatter: OppiaDateTimeFormatter
+) : ObservableViewModel() {
+
+  fun processDateAndTime(): String {
+    return oppiaDateTimeFormatter.formatDateFromDateString(
+      OppiaDateTimeFormatter.DD_MMM_hh_mm_aa,
+      eventLog.timestamp
+    )
+  }
+
+  fun processPriority(): String? {
+    return eventLog.priority.name.capitalizeWord()
+  }
+
+  fun processContext(): String? =
+    eventLog.context.activityContextCase.name.capitalizeWords().substringBeforeLast(' ')
+
+  fun processActionName(): String = eventLog.actionName.name.capitalizeWords()
+
+  private fun String.capitalizeWord(): String = toLowerCase().capitalize()
+
+  private fun String.capitalizeWords(): String =
+    toLowerCase().split("_").joinToString(" ") { it.capitalize() }
```

Comment:
This is getting used at two places but toLowerCase isn't required at one of them (context). We can keep this for splitting and capitalizing. Also, as suggested above -- `capitalize() --> replaceFirstChar(Char::uppercase)`

Reply:
Actually we need `toLowerCase` for `context` as well. See the original text below.
<img width="243" height="432" src="https://user-images.githubusercontent.com/63469151/124022593-a0f6f780-da0a-11eb-9195-59695fbedd2d.jpg">

Reply:
Aah, okay.